concrete/blocks/image/form.php

```diff
@@ -78,7 +78,36 @@
 </fieldset>
 
 <fieldset>
-    <legend><?php echo t('Resize Image'); ?></legend>
+    <legend><?php echo t('Alignment'); ?></legend>
+
+    <div class="form-group">
+        <div class="radio">
+            <label>
+                <?php
+                echo $form->radio('alignImage', 0, $alignImage);
+                echo t('Left');
+                ?>
+            </label>
+        </div>
+        <div class="radio">
+            <label>
+                <?php
+                echo $form->radio('alignImage', 1, $alignImage);
+                echo t('Center');
+                ?>
+            </label>
+        </div>
+        <div class="radio">
+            <label>
+                <?php
+                echo $form->radio('alignImage', 2, $alignImage);
+                echo t('Right');
+                ?>
+            </label>
+        </div>
+    </div>
+
+    <legend><?php echo t('Size'); ?></legend>
```

Comment:
It wasn't accidental. I figured that 'Image' was redundant because everything in the form is about the image. I further figured that the 'Re' was unnecessary because the subsequent option was clear enough, and I like pithy headings.

I'm happy to change it back if you think that's more concretey. If I were to do so, the prior heading should probably change from 'Alignment' to 'Image Alignment' for the sake of consistency.